salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -138,13 +138,18 @@
 
                 // specify allowed elements, attributes, and classes
                 // this must conform to the `STANDARD_MAPPING`
-                var filter = ' p em strong strike u sub sup; a[!href](salsah-link)';
+                var filter = ' p em strong strike u sub sup hr h1 h2 h3 h4 h5 h6 pre table tbody tr td ol ul li cite blockquote code; a[!href](salsah-link) ';
 
 				var config = {
 					language: (SALSAH.userprofile && SALSAH.userprofile.userData && SALSAH.userprofile.userData.lang) ? SALSAH.userprofile.userData.lang : 'en' ,
                     allowedContent: filter,
                     pasteFilter: filter,
+                    format_tags: 'p;h1;h2;h3;h4;h5;h6;pre',
```

Comment:
>default values h4, h5, h6
I don't understand this point...

I think `h4`, `h5` and `h6` should be default values anyway. How could that be  a `p`?

> idem, `h4` instead of `p`